ui/src/pages/Dashboard/NodeInfo/NodeAppHash.vue

```diff
@@ -185,8 +169,19 @@ export default {
       return this.pagination.perPage * (this.pagination.currentPage - 1);
     },
     total() {
-      this.paginationTotal(this.tableData.length);
-      return this.tableData.length;
+      let result;
+      if (this.searchQuery !== '') {
```

Comment:
avoid nesting. 
if (!this.searchQuery) {
return 'blabla'
}
.. continue with code here assuming we have searchQuery defined